src-web/actions/common.js

```diff
@@ -188,6 +188,17 @@ export const openModal = (...args) => {
   )
 }
 
+export const openActionModal = (namespace, resourceName, actionName, actionDescription) => {
+  // var restApi = '/kappnav/resource/kappnav/execute/command/'+actionName+'?namespace=kappnav'
```

Comment:
Remove commented out code

Reply:
I think the comment is there to show what the url would look like.

Reply:
If that's the case can we remove all of the variables?

Reply:
Well, I may have been wrong. The namespace definitely not be the same all the time.  I removed the comment.